pkg/pillar/cmd/zedmanager/handlevolumemgr.go

```diff
@@ -8,12 +8,11 @@ package zedmanager
 import (
 	"encoding/json"
 	"fmt"
-	"os"
-	"time"
-
 	"github.com/lf-edge/eve/pkg/pillar/base"
 	"github.com/lf-edge/eve/pkg/pillar/types"
 	uuid "github.com/satori/go.uuid"
+	"os"
+	"path"
```

Comment:
We changed to have a pattern with standard go import, blank line, other imports. Let's not change that back unless there is a very very good reason.

Reply:
oops, sorry. Will fix now